package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.99.0",
+  "version": "0.100.0",
```

Comment:
🎉 100 minors versions without a a full release 🙃

Reply:
What a milestone; here's to the next 100 ! 😆